src/main/java/hlf/java/rest/client/service/impl/AddOrgToChannelWriteSetBuilderImpl.java

```diff
@@ -177,34 +183,16 @@ private ConfigGroup setNewOrgGroup(String newOrgMspId) {
 
     return ConfigGroup.newBuilder()
         .setModPolicy(FabricClientConstants.CHANNEL_CONFIG_MOD_POLICY_ADMINS)
-        .putAllPolicies(setNewOrgPolicies(newOrgMspId))
+        .putAllPolicies(channelService.getDefaultRolePolicy(newOrgMspId))
         .putAllValues(valueMap)
-        .setVersion(0)
+        .setVersion(0) // First time update, hence version is 0
         .build();
   }
 
   private ConfigGroup setEmptyGroup(long version) {
     return ConfigGroup.newBuilder().setModPolicy("").setVersion(version).build();
   }
 
-  private Map<String, ConfigPolicy> setNewOrgPolicies(String newOrgName) {
-    Map<String, ConfigPolicy> applicationPoliciesMap = new HashMap<>();
-    applicationPoliciesMap.put(
-        FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_ADMINS,
-        setNewOrgPolicy(newOrgName, FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_ADMINS));
-    applicationPoliciesMap.put(
-        FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_ENDORSEMENT,
-        setNewOrgPolicy(newOrgName, FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_ENDORSEMENT));
-    applicationPoliciesMap.put(
-        FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_READERS,
-        setNewOrgPolicy(newOrgName, FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_READERS));
-    applicationPoliciesMap.put(
-        FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_WRITERS,
-        setNewOrgPolicy(newOrgName, FabricClientConstants.CHANNEL_CONFIG_POLICY_TYPE_WRITERS));
-
-    return applicationPoliciesMap;
-  }
-
   private ConfigPolicy setNewOrgPolicy(String newOrgName, String policyTarget) {
```

Comment:
[nitpick] `setNewOrgPolicy` `setTypeOnePolicy` is not used and can be deleted.

Reply:
Sure, Addressed.